internal/testbed/integration/prometheus/e2e_test.go

```diff
@@ -111,3 +115,44 @@ func installPrometheusNodeExporter() error {
 
 	return nil
 }
+
+func checkStartTimeStampPresent(t *testing.T, ms *consumertest.MetricsSink) {
+	// Skip the first two payloads, which may not have the start timestamp
+	// correctly set
+	require.True(t, len(ms.AllMetrics()) >= 2, "Expected at least 2 metrics payloads")
+
+	for i := 2; i < len(ms.AllMetrics()); i++ {
+		r := ms.AllMetrics()[i]
+
+		for _, rm := range r.ResourceMetrics().All() {
+			for _, sm := range rm.ScopeMetrics().All() {
+				for _, m := range sm.Metrics().All() {
+					switch m.Type() {
+					case pmetric.MetricTypeExponentialHistogram:
+						for _, dp := range m.ExponentialHistogram().DataPoints().All() {
+							assert.NotZero(t, dp.StartTimestamp(), "StartTimestamp should be non-zero for metric %s", m.Name())
```

Comment:
nit: let's put this error message into constant

Reply:
Good call. Done.